shared/js/content-scope/protections.js

```diff
@@ -24,7 +24,7 @@ export async function loadProtections () {
         'audio',
         'battery',
         'canvas',
-        'cookie',
```

Comment:
Can we use `-` dashes to make it consistent? Or change all the others to upper case?

Reply:
Alternatively we could have the keys as uppercase and the files as dasherised. I'm indifferent so long as it's consistent though :).